### DISCUSSION TITLE
能否同步不同的MaxKB知识库

### DISCUSSION THREAD

**Discussion by dlluckboy** [halo-dev/halo#6990] [Q&A]
### 你当前使用的版本

2.20.8

### 描述一下此特性

halo中可以添加MaxKB插件的功能很好，但是有个问题，上传的文章只能同步到MaxKB的一个知识库中，能否再上传的时候，可以选择不同的MaxKB知识库，这样方便管理。

**Comment by ruibaby** [comment]
ping @LIlGG

**Comment by LIlGG** [comment]
感谢提供的意见，不过当前 MaxKB 插件仅支持在发布文章时被动去监听发布事件。因此无法在上传时选择不同的知识库，只能通过预配置的方式。

**Reply by dlluckboy** [reply]
您说的这个预配置方式只的是什么？

**Reply by LIlGG** [reply]
> 您说的这个预配置方式只的是什么？

指的就是通过配置，预先设置应该同步至 MaxKB 的哪个知识库中。就是目前的实现方式。
